TheMovie/TheMovie/Common/Constants/Text.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Text.swift
+//  TheMovie
+//
+//  Created by Jeiel Lima on 15/03/24.
+//
+
+import Foundation
+
+struct Text {
+    struct Auth {
+        struct Login {
+            static let socialSignIn = "Sign in with social accounts"
+            static let or           = "or"
+            static let btnSignIn    = "Sign In"
+            static let createButton = "Create a Free Account"
+        }
+    }
+}
```

Comment:
Utilizar arquivo de Strings, através da abordagem Localizable

https://marceloproducts.medium.com/how-to-use-localizable-strings-in-xcode-with-swiftui-4aab75015c8